modules/cms/widgets/templatelist/partials/_items.htm

```diff
@@ -36,7 +36,7 @@ <h4><a href="javascript:;"><?= e($item->title) ?></a></h4>
 
                     <input type="hidden" name="template[<?= e($item->fileName) ?>]" value="0" />
                     <div class="checkbox custom-checkbox nolabel">
-                        <?php $cbId = 'cb'.md5($item->fileName) ?>
+                        <?php $cbId = 'cb'.md5($this->itemType . '/' . $item->fileName) ?>
```

Comment:
@wenlong-date Minor fix - would you mind adding some spacing around concatenated strings?
```suggestion
                        <?php $cbId = 'cb' . md5($this->itemType . '/' . $item->fileName) ?>
```

Reply:
surer! Thanks. I formatted code and had committed.